### PR TITLE
Jacoco coverage plugin

### PR DESCRIPTION
**Pull request by philipa**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -155,12 +167,6 @@
 		</exclusion>
 	  </exclusions>
     </dependency>
-    <dependency>
-      <groupId>org.hamcrest</groupId>
-      <artifactId>hamcrest-core</artifactId>
-      <version>1.2.1</version>
-      <scope>test</scope>
-    </dependency>
 	<dependency>
```
Comment (simschla) [inline pom.xml]: why did you need to remove this?

Comment (philipa) [inline pom.xml]: A later version is already included transitively via junit.

As per 4c3cc141867c62655c0b451044a4d3c54f2e5bd8:
"Rely on later version required by JUnit. The previous configuration caused Eclipse/m2e compilation path issues."

**Comment by ebrahim-moshaya** [comment]
Any idea when this will be merged?

**Comment by philipa** [comment]
I've been happily using this locally for 18 months with no problems. @simschla - any chance it will be merged?

**Comment by simschla** [comment]
I'll merge it and create a new version of the plugin!

**Comment by philipa** [comment]
Thanks Simon
